src/DefaultBuilder/src/BootstrapHostBuilder.cs

```diff
@@ -111,6 +111,11 @@ public IHostBuilder ConfigureServices(Action<HostBuilderContext, IServiceCollect
             };
 
             hostingEnvironment.ContentRootFileProvider = new PhysicalFileProvider(hostingEnvironment.ContentRootPath);
+            if (hostConfiguration[WebHostDefaults.WebRootKey] is string webRootPath)
+            {
+                hostingEnvironment.WebRootPath = ContentRootResolver.ResolvePath(hostingEnvironment.ContentRootPath, webRootPath);
```

Comment:
Why can't we unconditionally configure it the same way `ContentRootPath` is resolved?

Reply:
Also we assign the resolved path to `hostConfiguration[ContentRootPath]`. Do we need to do the same thing for web root?

Reply:
Also, `ContentRootResolver` becomes a misnomer with this name, I'd be open to a different name